cli/minimal/Makefile

```diff
@@ -0,0 +1,25 @@
+# **************************************************************************************
+#
+#									Very minimal makefile.
+#
+# **************************************************************************************
+
+EMULATOR = x65emu
```

Comment:
This may confuse people, as the emulator is named `emu` by default.

Maybe use one of the released binary names?
- emu_linux_arm64
- emu_linux_x64
- emu_windows_x64.exe